lib/Controller/SignalingController.php

```diff
@@ -146,18 +146,13 @@ public function getSettings(string $apiVersion, string $token = ''): DataRespons
 
 		$turn = [];
 		$turnSettings = $this->talkConfig->getTurnSettings();
-		if (!empty($turnSettings['server'])) {
-			$schemes = explode(',', $turnSettings['schemes']);
-			$protocols = explode(',', $turnSettings['protocols']);
-			foreach ($schemes as $scheme) {
-				foreach ($protocols as $proto) {
-					$turn[] = [
-						'url' => [$scheme . ':' . $turnSettings['server'] . '?transport=' . $proto],
-						'urls' => [$scheme . ':' . $turnSettings['server'] . '?transport=' . $proto],
-						'username' => $turnSettings['username'],
-						'credential' => $turnSettings['password'],
-					];
-				}
+		if (!empty($turnSettings)) {
+			foreach ($turnSettings as $setting) {
+				$turn[] = [
+					'urls' => $setting['urls'],
```

Comment:
can we keep the url thing for better compatibility with the mobile clients, or does it hurt somehow?

Reply:
I'd rather not keep it, because it can contain only one url, not multiple urls.

From https://developer.mozilla.org/en-US/docs/Web/API/RTCIceServer/url: _The obsolete RTCIceServer dictionary's url property specifies the URL of a single ICE server to be used while negotiating connections. It was removed from the specification in June 2013 but is still broadly used in older examples and books, so we include documentation here to help adapt old code to new browsers._

And unfortunately `url` should only contain _one_ element, not multiple elements as now in `urls`.

Reply:
Ah, this is the first set of duplication... okay fine by me